jsettlers.graphics/src/main/java/jsettlers/graphics/map/draw/settlerimages/SettlerImageMap.java

```diff
@@ -346,6 +346,16 @@ public Image getImageForSettler(EMovableType movableType, EMovableAction action,
 	 */
 	private SettlerImageMapItem getMapItem(EMovableType movableType,
 			EMovableAction action, EMaterialType material, EDirection direction) {
+
+
+		if(movableType==null||action==null||material==null||direction==null){
```

Comment:
use our formatter configuration to format the code. (spaces are missing between == and so on)